src/__tests__/CompareResults/RevisionSelect.test.tsx

```diff
@@ -27,8 +27,8 @@ describe('Revision select', () => {
     expect(await screen.findByTestId('revision-select')).toBeInTheDocument();
     expect(document.body).toMatchSnapshot();
 
-    const selectButton = await screen.findByRole('button', {
-      name: 'All revisions',
+    const selectButton = await screen.findByRole('combobox', {
+      name: 'Filter by revision',
```

Comment:
as you see, now the accessible name is more like a label, but before it was its value (I think because we didn't add an `aria-label`)